src/renderer/components/reader/ReaderOptions.tsx

```diff
@@ -245,7 +242,7 @@ export class ReaderOptions extends React.Component<Props> {
                             name="alignment"
                             type="radio"
                             onChange={(e) => this.props.handleSettingChange(e, "align", "left")}
-                            checked={this.props.settings.align === "left"}
+                            checked={settings.align === "left"}
```

Comment:
This data structure is actually strongly-typed, via `ReaderConfig` (for example, "left" is recognized as a valid enum value for `align`). So, whilst you are editing code in `ReaderOptions.tsx`, could you please change `settings: any;` to `settings: ReaderConfig;` in `interface Props extends TranslatorProps` at the top of the file? Thank you! (`import { ReaderConfig } from "readium-desktop/common/models/reader";`)

PS: Pierre is working in separate PRs for further removal of `any` and better typing across the board.